<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>36</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 22pt &quot;Arial&quot;;
color:  #132742;</string>
   </property>
   <property name="text">
    <string>Join the Team</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>81</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>INVITE CODE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E3E5E8;
font: 12pt &quot;Segoe UI&quot;;
border-radius: 8px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnJoin">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>178</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFCCCD;
font: 700 11pt &quot;Arial&quot;;
color: #132742;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Join Team</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>178</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Arial&quot;;
color: #132742;
background-color: #E3E5E8;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
